sw/device/tests/sim_dv/keymgr_sideload_kmac_test.c

```diff
@@ -84,6 +84,27 @@ static void test_kmac_with_sideloaded_key(dif_keymgr_t *keymgr,
   LOG_INFO("Computed KMAC output for sideloaded key.");
 
   CHECK_ARRAYS_EQ(output, (uint32_t *)sideload_digest_result, kKmacOutputLen);
+
+  LOG_INFO("Clearing the sideloaded key.");
+
+  // Enable "clear the key" toggle, so that previous sideload key port is
+  // cleared.
+  CHECK_DIF_OK(
+      dif_keymgr_sideload_clear_set_enabled(keymgr, kDifToggleEnabled));
+
+  // Disable "clear the key" toggle, so that the sideload key port is stable.
+  // Otherwise, the sideload port is continuously overwritten by fresh
+  // randomness every clock cycle.
+  CHECK_DIF_OK(
+      dif_keymgr_sideload_clear_set_enabled(keymgr, kDifToggleDisabled));
```

Comment:
Could this introduce a race condition? Not sure how keymgr handles a case where the clear bit is disabled before the cleared output has been written -- or maybe  this happens so quickly it isn't a concern?

Reply:
The randomization/clearing of the sideload slot happens only 2 clock cycles after keymgr HWIP receives the TL-UL bus write operation (for `SIDELOAD_CLEAR` register), so I think it is fast enough that we should not have race condition here.
![image](https://user-images.githubusercontent.com/7850700/204799038-a6dd7f64-d17b-4eff-890c-0ede49c53aca.png)